core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/aiservice/GuardrailsSupport.java

```diff
@@ -15,18 +9,28 @@
 import dev.langchain4j.rag.AugmentationResult;
 import io.quarkiverse.langchain4j.guardrails.GuardrailException;
 import io.quarkiverse.langchain4j.guardrails.InputGuardrail;
+import io.quarkiverse.langchain4j.guardrails.InputGuardrailResult;
 import io.quarkiverse.langchain4j.guardrails.OutputGuardrail;
+import jakarta.enterprise.inject.spi.CDI;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static dev.langchain4j.data.message.UserMessage.userMessage;
 
 public class GuardrailsSupport {
 
     public static void invokeInputGuardrails(AiServiceMethodCreateInfo methodCreateInfo, UserMessage userMessage,
             ChatMemory chatMemory, AugmentationResult augmentationResult) {
-        InputGuardRailsResult result = invokeInputGuardRails(methodCreateInfo,
-                new InputGuardrail.InputGuardrailParams(userMessage, chatMemory, augmentationResult));
-        if (!result.success()) {
-            throw new GuardrailException(
-                    "Input validation failed. The guardrail " + result.bean().getName() + " thrown an exception",
-                    result.failure());
+        InputGuardrailResult result = null;
+        try {
+            result = invokeInputGuardRails(methodCreateInfo,
+                    new InputGuardrail.InputGuardrailParams(userMessage, chatMemory, augmentationResult));
+        } catch (Exception e) {
+            throw new GuardrailException(e.getMessage(), e);
```

Comment:
Why do you have both validation exception and guardrail exception. I think the latter is enough

Reply:
This is necessary to cover the case when a `validate` implementation still throw a runtime exception like in [this case](https://github.com/quarkiverse/quarkus-langchain4j/pull/883/files#diff-20a556aa33caf2e1c7a85c82a63dfacc54d8cc0b88f31a615e37de63eed4c2a3R168).